invenio_collections/receivers.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, app=None):
             from .percolator import _find_matching_collections_externally
             self.matcher = _find_matching_collections_externally
 
-    def __call__(self, record, **kwargs):
+    def __call__(self, sender, record, **kwargs):
```

Comment:
As Travis says, you will also need to amend the tests correspondingly. But I want to know why is this change needed: can you point me to the signature of the signal?

Reply:
It's probably due to https://github.com/inveniosoftware/invenio-records/commit/cd3a7d2660e20f00063d142db32fba1ffec2fe6b (backward incompatible change)